_posts/2020-12-06-What-Kind-Of-Music-Can-I-Stream.md

```diff
@@ -3,7 +3,8 @@ layout: post
 title:  "What kind of music can I stream?"
 author: flavius
 categories: [ Music, DMCA, Audio ]
-image: assets/images/dmca'd.jpg
+image-location: "assets/images/"
```

Comment:
Move to `_config.yml`.

Reply:
Since this has been moved to _config.yml, do we still need this setting in the post?